docs/whats-new/2019-09.md

```diff
@@ -0,0 +1,160 @@
+- ***new!*** [Create gRPC client libraries](../architecture/grpc-for-wcf-developers/client-libraries.md) - gRPC e-book
+- ***new!*** [Create a new ASP.NET Core gRPC project](../architecture/grpc-for-wcf-developers/create-project.md) - gRPC e-book
+- ***new!*** [Create Docker images](../architecture/grpc-for-wcf-developers/docker.md) - gRPC e-book
+- ***new!*** [Encryption and network security](../architecture/grpc-for-wcf-developers/encryption.md) - gRPC e-book
+- ***new!*** [Error handling](../architecture/grpc-for-wcf-developers/error-handling.md) - gRPC e-book
+- ***new!*** [gRPC in production](../architecture/grpc-for-wcf-developers/grpc-in-production.md) - gRPC e-book
+- ***new!*** [gRPC overview](../architecture/grpc-for-wcf-developers/grpc-overview.md) - gRPC e-book
+- ***new!*** [ASP.NET Core gRPC for WCF Developers](../architecture/grpc-for-wcf-developers/index.md) - gRPC e-book
+- ***new!*** [Interface Definition Language](../architecture/grpc-for-wcf-developers/interface-definition-language.md) - gRPC e-book
+- ***new!*** [Introduction](../architecture/grpc-for-wcf-developers/introduction.md) - gRPC e-book
+- ***new!*** [Kubernetes](../architecture/grpc-for-wcf-developers/kubernetes.md) - gRPC e-book
+- ***new!*** [Load balancing gRPC](../architecture/grpc-for-wcf-developers/load-balancing.md) - gRPC e-book
+
+## .NET Core
+
+- ***new!*** [Breaking changes for migration from Version 2.2 to 3.0](../core/compatibility/2.2-3.0.md) - Breaking changes with markdown files
+- ***new!*** [Breaking change selectors](../core/compatibility/breaking-changes.md) - Breaking changes with markdown files
+- ***new!*** [CoreFx breaking changes](../core/compatibility/corefx.md) - Breaking changes: Filter by category, update TOC
+- ***new!*** [Cryptography breaking changes](../core/compatibility/cryptography.md) - Breaking changes: Filter by category, update TOC
+- ***new!*** [Globalization breaking changes](../core/compatibility/globalization.md) - Breaking changes: Filter by category, update TOC
+- ***new!*** [Visual Basic breaking changes](../core/compatibility/visualbasic.md) - Breaking changes: Filter by category, update TOC
+- ***new!*** [Breaking changes in Windows Forms (.NET Core to .NET Core)](../core/compatibility/winforms.md) - Breaking changes: Filter by category, update TOC
+- ***new!*** [Troubleshoot .NET Core tool usage issues](../core/tools/troubleshoot-usage-issues.md) - New article: troubleshoot dotnet tool usage
+- [.NET Core distribution packaging](../core/build/distribution-packaging.md) - Update package guidelines for .NET Core 3.0
+
+## C# language
+
+- [Compiler Error CS0034](../csharp/language-reference/compiler-messages/cs0034.md) - Some fixes in compiler-messages directory
+
+## .NET Core desktop
+
+- ***new!*** [Data binding overview in WPF](../desktop-wpf/data/data-binding-overview.md) - Desktop Guide - WPF Initial Release
```

Comment:
These are actually updates that got moved to the new Desktop guide. Should we remove the new labels?